client/src/routes/Session/components/Participants/Participant.tsx

```diff
@@ -90,10 +99,19 @@ const Participant: React.FC<ParticipantProps> = ({
   participant,
   topGradient,
 }) => {
+  const {call} = useContext(DailyContext);
   const {t} = useTranslation('Screen.Session');
   const photoURL = (participant?.userData as DailyUserData)?.photoURL;
   const theme = useExerciseTheme();
   const background = theme?.backgroundColor ?? COLORS.WHITE;
+  const isSessionHost = useIsSessionHost();
+
+  const onAudioToggle = useCallback(
+    (muted: boolean) => {
+      call?.updateParticipant(participant.user_id, {setAudio: !muted});
```

Comment:
Wow can anyone mute someone else? I guess this will change when we have proper auth tokens in daily :)